dataikuapi/dss/ml.py

```diff
@@ -1443,6 +1443,14 @@ def get_scatter_plots(self):
 
 
 class DSSMLTask(object):
+
+    @staticmethod
+    def from_full_model_id(client, fmi, project_key=None):
+        match = re.match(u'A-([\w]+)-([\w]+)-([\w]+)-s\w+-pp\w+-m\w+', fmi)
```

Comment:
Doesn't work with partitioned models, see `FullModelId.ANALYSIS_PATTERN`
```suggestion
        match = re.match(u'A-([\w]+)-([\w]+)-([\w]+)-s\d+-pp\d+(?:-base|-part-\w+)?-m\d+', fmi)
```
(suggestion needs to be tested)